modules/core/src/test/java/org/apache/ignite/internal/processors/cache/IgniteClientCacheStartFailoverTest.java

```
@@ -362,15 +360,6 @@ public void testRebalanceStateConcurrentStart() throws Exception {
                                 ", val=" + i + ']', e);
 
                             CacheConfiguration ccfg = cache.getConfiguration(CacheConfiguration.class);
-
```

It seems, that we swallow exception without failing test.